resources/definitions/3DI_D300_v2

```diff
@@ -0,0 +1,34 @@
+{
+    "version": 2,
+    "name": "3DI D300 v2",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "V27Jain",
+        "manufacturer": "3Deometry Innovations",
+        "file_formats": "text/x-gcode",
+        "platform": "3DI_D300_v2_platform.STL",
+        "has_materials": true,
+        "machine_extruder_trains":
+        {
+            "0": "3DI_D300_v2_extruder_0"
+        }
+    },
+
+    "overrides": {
+        "machine_center_is_zero": { "default_value": true },
+        "machine_depth": { "default_value": 310 },
```

Comment:
The specification, as far as I could find, specify that this printer is 300x300x300: https://3deometry.com/3d-printers/3deometry/3di-d300-3d-printer/

This is probably why it is called "D300" as well.